lxqt-config-input/resources/lxqt-config-touchpad-autostart.desktop.in

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=LXQt Touchpad Settings Loader
```

Comment:
For using weblate and yaml `Name` and `Comment` should be in a file `translations/lxqt-config-touchpad-autostart.desktop.yaml` if I remember well.
```
Desktop Entry/Name: "LXQt Touchpad Settings Loader "
Desktop Entry/Comment: "Autostart touchpad settings for lxqt-config-input"
```
But I think only `Name` is needed here, nothing else is shown in the GUI.

Reply:
Thanks for the review. Totally glossed over the YAML stuff.
Keeping the `Comment` section. It can be displayed in the tip, although as you stated, currently isn't.
Updating.